src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -4875,7 +4875,7 @@ components:
         - ERROR_CODE_MISSING_PRIVATE_PARAMS # The downloaded global configuration did not contain private parameters
         - ERROR_CODE_UNINITIALIZED # The configuration client is initializing.
         - UNKNOWN
-    CodeWithMetadata:
+    CodeWithDetails:
       type: object
       description: object that contains a code identifier and possibly collection of
```

Comment:
maybe add mention about validation errors here, now it mentions only code and metadata?

Reply:
Added a mention of validation errors